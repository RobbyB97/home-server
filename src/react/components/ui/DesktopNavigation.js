/* Libraries */
import React from 'react'


/* Component */
export const DesktopNavigation = () => (
    <div id="DesktopNavigation">
        <ul>
            <li>
                <a href="#about">about</a>
            </li>
            <li>
                <a href="#services">services</a>
            </li>
            <li>
                <a href="#skills">skills</a>
            </li>
            <li>
                <a href="#experience">experience</a>
            </li>
            <li>
                <a href="#projects">projects</a>
            </li>
        </ul>
    </div>
)


export default DesktopNavigation