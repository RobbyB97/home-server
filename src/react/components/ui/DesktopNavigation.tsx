export {}

/* Libraries */
import * as React from 'react'


/* Component */
export class DesktopNavigation extends React.Component<{}, {}> {
    public render() {
        return (
            <div id="DesktopNavigation">
                <ul>
                    <li>
                        <a href="#about">about</a>
                    </li>
                    <li>
                        <a href="#services">services</a>
                    </li>
                    <li>
                        <a href="#skills">skills</a>
                    </li>
                    <li>
                        <a href="#experience">experience</a>
                    </li>
                    <li>
                        <a href="#projects">projects</a>
                    </li>
                </ul>
            </div>
        )
    }
}


export default DesktopNavigation;