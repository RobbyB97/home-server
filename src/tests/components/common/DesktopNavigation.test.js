import React from 'react'
import {shallow} from 'enzyme'

import DesktopNavigation from '../../../react/components/common/DesktopNavigation'


test('Render DesktopNavigation', () => {
    const wrapper = shallow(<DesktopNavigation />)
    
    expect(wrapper).toMatchSnapshot()
})