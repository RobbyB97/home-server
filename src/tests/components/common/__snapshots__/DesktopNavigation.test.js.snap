// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render DesktopNavigation 1`] = `
<div
  id="desktop-navigation"
>
  <ul>
    <li>
      <NavLink
        activeClassName="active"
        ariaCurrent="true"
        exact={true}
        to="/"
      >
        Home
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="active"
        ariaCurrent="true"
        to="/skills/"
      >
        Skills
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="active"
        ariaCurrent="true"
        to="/experience/"
      >
        Experience
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="active"
        ariaCurrent="true"
        to="/projects/"
      >
        Projects
      </NavLink>
    </li>
    <li>
      <a
        href="mailto:bergersr@my.easternct.edu"
      >
        Contact
      </a>
    </li>
    <li
      className="desktop-navigation__link github"
    >
      <a
        href="https://www.github.com/RobbyB97"
        target="_blank"
      >
        <div />
      </a>
    </li>
    <li
      className="desktop-navigation__link linkedin"
    >
      <a
        href="https://www.linkedin.com/in/robbyb97"
        target="_blank"
      >
        <div />
      </a>
    </li>
  </ul>
</div>
`;
